.github/workflows/build-push-greenhouse-image.yaml

```diff
@@ -154,3 +145,13 @@ jobs:
         # This step uses the identity token to provision an ephemeral certificate
         # against the sigstore community Fulcio instance.
         run: echo "${TAGS}" | xargs -I {} cosign sign --yes {}@${DIGEST}
+
+  notify-on-failure:
+    if: failure()
+    needs: [build-and-push-greenhouse-image]
+    uses: cloudoperators/juno/.github/workflows/shared-slack-notification.yaml@main
+    with:
+      title: "🚨 JUNO Supernova Image Failed 🚨"
```

Comment:
Same here, Greenhouse instead of Supernova?

Reply:
Thx! @hodanoori